### DISCUSSION TITLE
In page nav and tabs visual representation

### DISCUSSION THREAD

**Discussion by marektonik** [twilio-labs/paste#3368] [General] [answered]
hey! I'm working on the Evergreen -> Paste design migration. I now that we have 2 components for different purpose: `in page nav` and `tabs`. It's quite easy to recognize them in the horizontal layout, however tabs in vertical look like in page nav. I have a case were we need to use In page nav and vertical tabs together and it's not looking really accessible. What do you think about my case and in general about tabs difference in style between horizontal and vertical?

In page nav vs. tabs
<img width="638" alt="Zrzut ekranu 2023-08-2 o 08 35 08" src="https://github.com/twilio-labs/paste/assets/140410476/8acd10e5-b4e2-4d63-831f-20b477835263">

My case:
<img width="1070" alt="Zrzut ekranu 2023-08-2 o 08 28 44" src="https://github.com/twilio-labs/paste/assets/140410476/0ac71450-5d4c-4832-a409-c7dc2ceb53de">

**Comment by loreina** [comment, marked as answer]
Hi @marektonik,

We have a ticket in our backlog to add a vertical variant to In Page Navigation. As part of that work, we may revise the design of the vertical variant of Tabs since there's some visual overlap. 

I'd recommend moving forward with the designs as-is since our components go through a peer review process which means it's approved by design teams for their products. Any updated design for vertical Tabs will push up to Figma files and code when that's released. Thanks!
